src/contrib/testkits/Akka.TestKit.Xunit2/TestKit.cs

```diff
@@ -21,6 +21,38 @@ namespace Akka.TestKit.Xunit2
     /// </summary>
     public class TestKit : TestKitBase , IDisposable
     {
+        private class PrefixedOutput : ITestOutputHelper
+        {
+            private readonly ITestOutputHelper output;
+            private readonly string prefix;
+
+            public PrefixedOutput(ITestOutputHelper output, string prefix)
+            {
+                this.output = output;
+                this.prefix = prefix;
+            }
+
+            public void WriteLine(string message)
+            {
+                output.WriteLine(prefix + message);
+            }
+
+            public void WriteLine(string format, params object[] args)
+            {
+                output.WriteLine(prefix + format, args);
+            }
+        }
+
+        private void InitializeLogger(ActorSystem system, ITestOutputHelper output)
```

Comment:
it is being used in some unit tests. The reason for this method is in case, there are multiple akka systems started in one test, the output will have only log messages from the "main" system. But log messages from all other systems are lost. This method will initiate the test log output also for additional system with a `prefix` for each log message

Reply:
right, was looking at the wrong line